templates/stable-diffusion-pretraining/03_end_to_end.ipynb

```diff
@@ -61,22 +66,9 @@
     "| 10 | Output path | Change to a permanent path |\n",
     "| 11 | Run the process script | Run as an Anyscale Job |\n",
     "| 12 | First phase of training | Run for 100 epochs |\n",
```

Comment:
Oh Actually it's not 100 epochs but 550,000 steps for resolution 256, and 850,000 steps for resolution 512. Approximately 1~2 epochs.

Reply:
ok I will update accordingly